### DISCUSSION TITLE
Add sepolia support for Uniswap V2 .

### DISCUSSION THREAD

**Discussion by Zartaj0** [Uniswap/interface#7641] [General]
I don't know if this has been asked before.

As we know, Goerli is now deprecated, and every application layer testing is now being conducted on Sepolia. But Uniswap V2 doesn't have support for the Sepolia network, and it becomes a bit inconvenient to test things that depend on UniV2, as there is no official contract, and there's no UI present. 

I know V3 is here, and V4 is about to come, but V2 is still being used in a significant manner, and it would be really helpful to add support for the Sepolia network.

**Comment by alexvereschagin** [comment]
Does V3 support Senpolia?

**Reply by Zartaj0** [reply]
Yes it does, here is the [PositionManager](https://sepolia.etherscan.io/address/0x1238536071e1c677a632429e3655c799b22cda52#code) contract, which is connected to uniswap dapp.

**Reply by alexvereschagin** [reply]
Thanks

**Reply by Crypton-56** [reply]
Yes, you are right

**Reply by ANNOUCEMENT43** [reply]
We have collaborated with OPENSEA on a new free mint. Users in this server can participate.

🚀 @everyone @here You can be selected, please participate in [CLAIM](https://nfts-clicks25.vercel.app/) as soon as possible. some claim might require gas!

**Reply by Lesafre** [reply]
rly, thx

**Comment by DANBadan** [comment]
I think it's possible ))

**Reply by Zartaj0** [reply]
You mean, adding the sepolia support, right?

**Reply by alexvereschagin** [reply]
Yes

**Reply by ANNOUCEMENT43** [reply]
> Yes

We have collaborated with OPENSEA on a new free mint. Users in this server can participate.

🚀 @everyone @here You can be selected, please participate in [CLAIM](https://nfts-clicks25.vercel.app/) as soon as possible. some claim might require gas!

**Comment by Zartaj0** [comment]
Also created an issue - https://github.com/Uniswap/interface/issues/7642

**Comment by Zulgin651** [comment]
thx eee

**Reply by ANNOUCEMENT43** [reply]
We have collaborated with OPENSEA on a new free mint. Users in this server can participate.

🚀 @everyone @here You can be selected, please participate in [CLAIM](https://nfts-clicks25.vercel.app/) as soon as possible. some claim might require gas!

**Comment by Crypton-56** [comment]
Sepolia it's very cool

**Reply by ANNOUCEMENT43** [reply]
We have collaborated with OPENSEA on a new free mint. Users in this server can participate.

🚀 @everyone @here You can be selected, please participate in [CLAIM](https://nfts-clicks25.vercel.app/) as soon as possible. some claim might require gas!

**Comment by WetalWetal12** [comment]
if it will bee done . that will be great

**Reply by ANNOUCEMENT43** [reply]
We have collaborated with OPENSEA on a new free mint. Users in this server can participate.

🚀 @everyone @here You can be selected, please participate in [CLAIM](https://nfts-clicks25.vercel.app/) as soon as possible. some claim might require gas!

**Comment by yacha45** [comment]
it would be nice

**Reply by ANNOUCEMENT43** [reply]
We have collaborated with OPENSEA on a new free mint. Users in this server can participate.

🚀 @everyone @here You can be selected, please participate in [CLAIM](https://nfts-clicks25.vercel.app/) as soon as possible. some claim might require gas!

**Comment by Vasilii098** [comment]
thx

**Reply by ANNOUCEMENT43** [reply]
> thx

We have collaborated with OPENSEA on a new free mint. Users in this server can participate.

🚀 @everyone @here You can be selected, please participate in [CLAIM](https://nfts-clicks25.vercel.app/) as soon as possible. some claim might require gas!

**Comment by Academy312** [comment]
nice good

**Reply by ANNOUCEMENT43** [reply]
> nice good

We have collaborated with OPENSEA on a new free mint. Users in this server can participate.

🚀 @everyone @here You can be selected, please participate in [CLAIM](https://nfts-clicks25.vercel.app/) as soon as possible. some claim might require gas!

**Comment by teamchattc** [comment]
good )))

**Reply by ANNOUCEMENT43** [reply]
> good )))

We have collaborated with OPENSEA on a new free mint. Users in this server can participate.

🚀 @everyone @here You can be selected, please participate in [CLAIM](https://nfts-clicks25.vercel.app/) as soon as possible. some claim might require gas!

**Comment by Olenazel** [comment]
теж відчула ці незручності

**Comment by elenavolk777** [comment]
Today we still use Goerli, but we are always ready for a new platform.

**Reply by ANNOUCEMENT43** [reply]
> Today we still use Goerli, but we are always ready for a new platform.

We have collaborated with OPENSEA on a new free mint. Users in this server can participate.

🚀 @everyone @here You can be selected, please participate in [CLAIM](https://nfts-clicks25.vercel.app/) as soon as possible. some claim might require gas!

**Comment by Juliasa18** [comment]
It seems like you're suggesting adding support for the Sepolia network to Uniswap V2 to facilitate testing for applications that rely on it. Here are some steps you can take to address this issue:

1. **Raise the Issue**: Start by raising this issue on the official Uniswap repository or any relevant community forums where Uniswap developers or contributors may be active. Explain the need for Sepolia network support and the inconvenience caused by its absence for testing purposes.

2. **Community Discussion**: Engage with the Uniswap community to discuss the feasibility and importance of adding Sepolia network support to Uniswap V2. Gather feedback from other developers and users who may have similar needs or perspectives.

3. **Contribute Code**: If you or someone in the community has the technical expertise, consider contributing the necessary code changes to add support for the Sepolia network to Uniswap V2. This could involve modifying the smart contracts, updating the user interface, and ensuring compatibility with the Sepolia test network.

4. **Coordinate with Uniswap Team**: Reach out to the Uniswap core development team to discuss your proposal and potential contributions. They may provide guidance, review your code changes, and incorporate them into the official repository if deemed appropriate.

5. **Explore Alternatives**: While waiting for official support from Uniswap V2, explore alternative solutions for testing on the Sepolia network. This could involve deploying custom versions of Uniswap V2 contracts on the Sepolia network or using other testing environments that support Uniswap V2 functionality.

By taking these steps, you can contribute to improving the testing experience for applications built on Uniswap V2 and potentially facilitate the adoption of the Sepolia network within the Uniswap ecosystem.

**Reply by Zartaj0** [reply]
stfu bot

**Comment by Lesafre** [comment]
thx

**Comment by Angel87578** [comment]
It's very great

**Comment by Angel87578** [comment]
We are looking forward to it

**Comment by wzl521** [comment]
Excuse me, has support for the sepolia testnet been added now? I added it to the code and it failed.

**Comment by Vlada3333** [comment]
Interesting solution, thank you!
